dev.jonmagon.kdiskmark.yaml

```
@@ -0,0 +1,53 @@
+app-id: dev.jonmagon.kdiskmark
+runtime: org.kde.Platform
+runtime-version: '5.15-21.08'
+sdk: org.kde.Sdk
+rename-desktop-file: kdiskmark.desktop
+rename-icon: kdiskmark
+copy-icon: true
+command: kdiskmark
+finish-args:
+  - --socket=x11
+  - --socket=wayland
+  - --filesystem=host
+  - --device=dri
+modules:
+  - name: kdiskmark
+    buildsystem: cmake-ninja
+    config-opts:
+      - -DCMAKE_BUILD_TYPE=Release
+      - -DROOT_EDITION=OFF
+    sources:
+      - type: git
+        url: https://github.com/JonMagon/KDiskMark.git
+        tag: 3.1.2-standalone
+        commit: b411c4ae0d4a4ab23531ae1fecddcf76a493a515
+        x-checker-data:
+          type: git
+    modules:
+      - name: fio
+        buildsystem: simple
+        build-commands:
+          - ./configure --prefix=$FLATPAK_DEST
+          - make -j $FLATPAK_BUILDER_N_JOBS
+          - make install
+        sources:
+          - type: git
+            url: https://github.com/axboe/fio.git
+            tag: fio-3.32
+            commit: db7fc8d864dc4fb607a0379333a0db60431bd649
+            x-checker-data:
+              type: git
+        modules:
+          - name: libaio
+            buildsystem: simple
+            build-commands:
+              - make prefix=$FLATPAK_DEST install
+            sources:
+              - type: git
+                url: https://pagure.io/libaio.git
+                tag: libaio-0.3.113
+                commit: 1b18bfafc6a2f7b9fa2c6be77a95afed8b7be448
+                x-checker-data:
+                type: git
```

```suggestion
                x-checker-data:
                    type: git
```

it's yaml, indentation is significant.